elements/cloud-init-growpart-lvm/README.rst

```diff
@@ -0,0 +1,11 @@
+=============
+cloud-init-growpart-lvm
```

Comment:
Does growpart increase the size of the PV? We have a growroot.yml playbook in kayobe-config that finds the root VG, and grows the partitions containing the PVs in it, then runs `pvresize` on the PVs.

Even after that, the LVs are still small, and we use kayobe to grow them.

It's not clear to be how suitable this LVM image is for general use machines without all of that automation on top.

Reply:
This work is related to SMS Lab instances, to make them more ,,user-friendly''. Let's not mix it with Kayobe usage.

Reply:
But I agree we should make pvresize automatic.

Reply:
I thought that this work started out as building images for for the kayobe multi-node test environments.

Surely the most user friendly option would be to not use LVM at all?

Reply:
From one perspective - yes, but do we want to go back to building both lvm and non-lvm images in SMS Lab? I doubt that.

Reply:
growpart on rhel family systems does not have pvresize in the code, on ubuntu it does

Reply:
I've created a bug for that:
https://bugzilla.redhat.com/show_bug.cgi?id=2122575

Reply:
Yes, I suggest we build separate LVM and non-LVM images. LVM for Kayobe (medium term we should be using the kayobe images for this rather than rebuilding) non-LVM for everyone else.

Who wants to have to expand their 1G /tmp LV after booting an instance?

Reply:
I will include non-lvm image builds in our CI:
https://stackhpc.atlassian.net/browse/DEV-1044

Reply:
:+1: